### ISSUE TITLE
deno fmt ignore top level await

### ISSUE THREAD

**Issue by keroxp** [denoland/deno#3883]
New version of `deno fmt` can't parse code taht contains top level await.

Case1: eliminated
<img width="396" alt="スクリーンショット 2020-02-04 23 47 36" src="https://user-images.githubusercontent.com/927286/73756484-f940b000-47aa-11ea-8973-a14da304e0e4.png">

Case2: failed to parse

```ts
async function f() {
    
}
await f();
```

```
$ deno fmt *
Error formatting: tla.ts
   Line 4, column 7: Expected ';', '}' or <eof>
Formatted 0 files in 884.48µs
```

**Comment by nayeemrmn** [comment]
cc @dsherret

**Comment by dsherret** [comment]
Thanks! I opened an issue in swc and it should be easy to get this supported.

Anyway we can transfer these issues to https://github.com/dsherret/dprint/issues?

**Comment by bartlomieju** [comment]
> Thanks! I opened an issue in swc and it should be easy to get this supported.
> 
> Anyway we can transfer these issues to https://github.com/dsherret/dprint/issues?

Looks like one can only transfer issues between repos you're collaborator in

**Comment by dsherret** [comment]
@bartlomieju ok, not a big deal.

Opened https://github.com/dsherret/dprint/issues/88

**Comment by dsherret** [comment]
This is fixed in dprint 0.4.3 along with various other improvements and fixes.

Next improvement will be getting the remainder of TS 3.8 features in: https://github.com/dsherret/dprint/milestone/1
